pkg/azureclients/armclient/azure_armclient.go

```diff
@@ -55,6 +80,8 @@ type Client struct {
 func New(authorizer autorest.Authorizer, clientConfig azureclients.ClientConfig, baseURI, apiVersion string, sendDecoraters ...autorest.SendDecorator) *Client {
 	restClient := autorest.NewClientWithUserAgent(clientConfig.UserAgent)
 	restClient.Authorizer = authorizer
+	restClient.Sender = getSender()
```

Comment:
This isn't a complete revert of kubernetes-sigs/cloud-provider-azure#1567 since it preserves the centralized retry and error handling. It seems to simply restore the previous customized common transport.

Reply:
I tested that not having the common transport was the issue and not the centralized retries.

Reply:
I only reverted the changes that affected the performance.